src/index.js

```diff
@@ -21,7 +21,7 @@ require('typeface-roboto');
 const history = createHistory();
 const router = routerMiddleware(history);
 
-const webSocket = new WebSocket('ws://localhost:8000/ws');
+const webSocket = new WebSocket('ws://localhost:8008/ws');
```

Comment:
Not something to do now, but we need to work out how to make this a configuration setting (and ultimately part of the url so we can point at different places e.g. the test server, the simulator, or different panda boxes).